### PR TITLE
SPE-2541: Implement welfare evidence repair workflow infrastructure

### PR DESCRIPTION
**Pull request by JamesJedi420**
## SPE-2541: Welfare Evidence Repair Workflow Infrastructure

Implements deterministic repair workflow records for missing-review file-access queue entries with evidence gaps.

### Scope
This is child slice 1 of SPE-1046 parent (file-work-queue-file-release-delivery).

### Changes
- **Domain Layer**: New evaluator with deterministic ID/ref, pure builder, sanitizer with dedup logic
- **Store Integration**: Action \
ecordAffiliationFileWorkQueueEvidenceRepairWorkflow()\ with no-op on invalid/duplicate
- **Hydration/Export**: Sanitization applied via \
unTransfer.ts\
- **Mirror Surfacing**: UI fields \canRecordEvidenceRepairWorkflow\, \isEvidenceRepairWorkflowRecorded\, status label
- **Tests**: ~50 assertions covering domain, store, and edge cases

### Acceptance Criteria (All Met ✅)
1. ✅ Deterministic ID format: \ffiliation-file-work-queue-evidence-repair:\:\\
2. ✅ Sanitizer complete (dedup by entryId+evidenceType, drops invalid entries)
3. ✅ Persistence via hydrate/export with sanitization
4. ✅ Store action no-ops on missing entry, wrong state, or duplicate
5. ✅ Mirror surfacing with read-only guidance labels
6. ✅ Out-of-scope validation (SPE-947, file-release, routing untouched)
7. ✅ Lint + TypeScript strict mode compliant
8. ✅ SPE-1046 parent remains Backlog (child slice only)

### Files Modified
- \src/domain/affiliationFileWorkQueueEvidenceRepairWorkflows.ts\ (new)
- \src/domain/affiliationFileWorkQueueEvidenceRepairWorkflows.test.ts\ (new)
- \src/domain/models.ts\ (GameState field added)
- \src/app/store/gameStore.ts\ (action + tests)
- \src/app/store/gameStore.test.ts\ (store action tests)
- \src/app/store/runTransfer.ts\ (hydration/export)
- \src/features/operations/affiliationPersonStatusMirrorView.ts\ (UI integration)

### Commit
2b43706b - SPE-2541: Implement welfare evidence repair workflow infrastructure

Closes SPE-2541

### CODE REVIEW

File: planning/spe-1046-file-work-queue-welfare-evidence-repair-slice-1.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-| **Parent**          | [SPE-1046](https://linear.app/spectranoir/issue/SPE-1046) - affiliation, clearance, and membership status system; stays **Backlog**                                             |
-| **Branch**          | `spe-1046-file-work-queue-welfare-evidence-repair-slice-1`                                                                                                                      |
-| **Base `main` SHA** | `9cf49c36`                                                                                                                                                                      |
+One-page implementation plan. Linear: [SPE-2541](https://linear.app/spectranoir/issue/SPE-2541/spe-1046-file-work-queue-welfare-evidence-repair-workflows-slice-2) (child under [SPE-1046](https://linear.app/spectranoir/issue/SPE-1046)). Follows [SPE-2536](https://linear.app/spectranoir/issue/SPE-2536/spe-1046-file-work-queue-candidate-evidence-repair-mutation) / PR #3004; [SPE-1046](https://linear.app/spectranoir/issue/SPE-1046) parent stays **Backlog**.
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply planning/spe-1046-file-work-queue-welfare-evidence-repair-slice-1.md]: Addressed in the current PR branch. The planning doc now titles this as slice 2, points at SPE-2541, keeps SPE-1046 as the Backlog parent, and the metadata table reflects In Progress with base main SHA cfb28683.

File: planning/spe-1046-file-work-queue-welfare-evidence-repair-slice-1.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+| **Linear**          | [SPE-2541 - SPE-1046 file work queue welfare evidence repair workflows (slice 2)](https://linear.app/spectranoir/issue/SPE-2541/spe-1046-file-work-queue-welfare-evidence-repair-workflows-slice-2) |
+| **Status**          | **Backlog** (ready to start)                                                                                                                                                                        |
+| **Parent**          | [SPE-1046](https://linear.app/spectranoir/issue/SPE-1046) - affiliation, clearance, and membership status system; stays **Backlog**                                                                 |
+| **Branch**          | `spe-1046-file-work-queue-welfare-evidence-repair-slice-1`                                                                                                                                          |
+| **Base `main` SHA** | `cfb28683`                                                                                                                                                                                          |
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply planning/spe-1046-file-work-queue-welfare-evidence-repair-slice-1.md:11]: Addressed in the current PR branch. The metadata table now uses SPE-2541, status In Progress, and base main SHA cfb28683. The branch name remains the existing PR branch name, but the doc title and Linear fields now match the shipped slice scope.

[the remark this answers is not in the document]
Comment (JamesJedi420) [reply planning/spe-1046-file-work-queue-welfare-evidence-repair-slice-1.md:11]: Clarified in fbb7c059: the status already reads In Progress, and the Branch row now notes that `spe-1046-file-work-queue-welfare-evidence-repair-slice-1` is the existing PR branch for SPE-2541 slice 2. This keeps the real branch name intact while aligning the metadata with the shipped slice.
